actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -29,7 +29,25 @@ def channel_class_names
           Pathname.new(channel_path).basename.to_s.split('.').first.camelize
         end
       end
+
+      # Returns constant of subscription adapter specified in config/cable.yml.
+      # If the adapter cannot be found, this will default to the Redis adapter.
+      # Also makes sure proper dependencies are required.
+      def pubsub_adapter
+        adapter = (cable.fetch('adapter') { 'redis' })
```

Comment:
I understand there's a performance win when there's some actual work done in the block, but when that's not the case, doesn't take the block any memory?